File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/DurationField.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class DurationField 
{
    public string Name { get; set; }
    public DurationFieldType Type { get; set; }
    public bool Supported { get; set; }
    public bool Precise { get; set; }
    public long UnitMillis { get; set; }
}

File: server-scala-play-server/app/model/DurationField.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for DurationField.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class DurationField(
  name: Option[String],
  `type`: Option[DurationFieldType],
  supported: Option[Boolean],
  precise: Option[Boolean],
  unitMillis: Option[Long]
)

object DurationField {
  implicit lazy val durationFieldJsonFormat: Format[DurationField] = Json.format[DurationField]
}

